cspell.json

```diff
@@ -6,7 +6,8 @@
 		"CHANGELOG.md",
 		"lib",
 		"node_modules",
-		"pnpm-lock.yaml"
+		"pnpm-lock.yaml",
+		"./src/bin/__snapshots__"
```

Comment:
I'd rather just have the snapshot be inline. Less configuration to work with.

Reply:
I can do this of course, but are you sure? Having added all the logic to automate the creation of the help output from the  `argOptions` it feels like a retrograde step to introduce a fully hard-coded string in the test that will have to be edited every time there is a change. Whereas the automatic snapshot file can be updated by pressing `u` in the test runner.

Reply:
Inline snapshots are updated by pressing `u` too 🙂 otherwise I'd absolutely want the snapshot file.

Reply:
Ah, great! For some reason I assumed that wouldn't be possible 😧 now I see why inline snapshots are much better! I've swapped it out for an inline one.